### PR TITLE
#513 chore: Add openapi generator requirements overrides

### LINKED ISSUE #513: ImportError: cannot import name 'escape' from 'jinja2'
Branch: 
feature/248

Steps:
1. `harness-application backend-example -t webapp`
2. Install ch and backend-example requirements
3. Run \__main\__.py

Expected:
 Server starts

Result:
>ImportError: cannot import name 'escape' from 'jinja2' 

SO reference: https://stackoverflow.com/questions/71718167/importerror-cannot-import-name-escape-from-jinja2

### PR DESCRIPTION
**Pull request by afonsobspinto**
Closes #513

Implemented solution: Overrides openapi-generator requirements.txt with missing requirements

How to test this PR: 
- Follow the steps described in the issue.
- Server should start instead of giving the error

### Sanity checks:
- [x] The pull request is explicitly linked to the relevant issue(s)
- [x] The issue is well described: clearly states the problem and the general proposed solution(s)
- [x] From the issue and the current PR it is explicitly stated how to test the current change
- [x] The labels in the issue set the scope and the type of issue (bug, feature, etc.)
- [ ] All the automated test checks are passing
- [ ] All the linked issues are included in one milestone
- [ ] All the linked issues are in the Review/QA column of the [board](https://app.zenhub.com/workspaces/cloud-harness-5fdb203b7e195b0015a273d7/board)
- [x] All the linked issues are assigned

### Breaking changes (select one):
- [x] The present changes do not change the preexisting api in any way
- [ ] This PR and the issue are tagged as a `breaking-change`

### Possible deployment updates issues (select one):
- [x] There is no reason why deployments based on CloudHarness may break after the current update
- [ ] This PR and the issue are tagged as `alert:deployment`

### Test coverage (select one):
- [ ] Tests for the relevant cases are included in this pr
- [x] The changes included in this pr are out of the current test coverage scope

### Documentation (select one):
- [ ] The documentation has been updated to match the current changes
- [x] The changes included in this PR are out of the current documentation scope

### CODE REVIEW

File: tools/harness-application [hunk 1]
```diff
@@ -54,8 +55,12 @@ if __name__ == "__main__":
 
     app_path = os.path.join(APPLICATIONS_SRC_PATH, args.name)
     for template_name in args.templates:
+
         if template_name == 'server':
-            generate_server(app_path)
+            with tempfile.TemporaryDirectory() as tmp_dirname:
+                copymergedir(os.path.join(HERE, APPLICATION_TEMPLATE_PATH, template_name), tmp_dirname)
+                merge_configuration_directories(app_path, tmp_dirname)
+                generate_server(app_path, tmp_dirname)
```
Comment (filippomc) [inline tools/harness-application:63]: Have to remove the tmp path at the end (suggest to use try...finally to ensure it is deleted)

**Review by filippomc — COMMENTED** [review]
Looks good, just a minor comment to address

File: tools/harness-application (hunk 1 above)
Comment (afonsobspinto) [reply tools/harness-application:63]: This should be happening automatically because I'm creating the temp folder using the context manager *with*

Comment (filippomc) [reply tools/harness-application:63]: right, nice!

**Review by filippomc — APPROVED** [review]
Looks good to me

**Pull request merged**